appveyor.yml

```diff
@@ -3,6 +3,9 @@ build: off
 cache:
   - 'C:\\tmp'
 environment:
+  global:
+    # Please, change to use token of PyCQA/pylint
+    COVERALLS_REPO_TOKEN: "R8noz2lfRpFXNPKQirwNHJyLl1wMIJnYF"
```

Comment:
@PCManticore
Could you send me the repo token?
![repo_token](https://cloud.githubusercontent.com/assets/6644187/14569193/c0b1fc62-0302-11e6-9622-adbfd77da2f2.png)

To add it to appveyor and got other build coveralls.io from windows